apps/catalogue3/src/components/detailView/EmailFieldValue.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <a :href="'mailto:' + data">{{ data }}</a>
```

Comment:
why not leave it up to the data input , by setting the correct url the DM can choose to add a link or mailto or something else

Reply:
we have a field of type=email and therefore we have this